panel.js

```diff
@@ -72,6 +65,59 @@ async function load() {
   $('#cards').html(outerdiv);
 }
 
+async function migrateFavourites() {
+  // migrate favourites (from cards, giftcards, ibans) to new format
+  // this function can be eventually removed in future versions
```

Comment:
Not tested logic: If this fails somehow, should we let the users start with a clean (empty0 state? try catch?

Reply:
Good point, I have added a try-catch in the function. If there is an error the extension `load` will continue using the extension data (clean start)